cmd/policy-main.go

```diff
@@ -320,6 +320,10 @@ func runPolicyLinksCmd(ctx *cli.Context) {
 				continue
 			}
 
+			if content.Type.IsDir() && isRecursive {
```

Comment:
In-case of non-recursive we would want to show.. if there are other directories available for navigation. It is expected..